language/move-analyzer/src/symbols.rs

```diff
@@ -373,9 +373,11 @@ impl SymbolicatorRunner {
                         // cases when developer indeed intended to open a standalone file that was
                         // not meant to compile
                         missing_manifests.insert(starting_path);
-                        if let Err(err) =
-                            sender.send(Err(anyhow!("Unable to find package manifest")))
-                        {
+                        if let Err(err) = sender.send(Err(anyhow!(
+                            "Unable to find package manifest. Make sure that
```

Comment:
I think you want `\` at the end of these lines? Otherwise it will print it out with new lines (I think)

Reply:
Interestingly, I checked this before sending the PR and the display is unaffected.